products/oneplus.md

```diff
@@ -71,6 +72,7 @@ releases:
 
 OnePlus phones run OxygenOS, which is based on Android. It receives updates every two months.
 
+The OnePlus 13 & 13R supports [five major Android updates](https://community.oneplus.com/thread/1809805181760569353) and 6 years of security updates.  
```

Comment:
```suggestion
The OnePlus 13 & 13R supports [four major Android updates](https://community.oneplus.com/thread/1809805181760569353) and six years of security updates.  
```

As per https://community.oneplus.com/thread/1809805181760569353 (four major Android updates => the device will see 5 major android versions).